docs/workflows/account-setup.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Account Setup
-sidebar_position: 2
+sidebar_position: 0
```

Comment:
Are we still planning to use an intro workflow from Colton?

Reply:
No, we're not going to have an intro anymore